shell/pom.xml

```diff
@@ -38,22 +38,79 @@
     <!--library versions -->
     <commons.lang3.version>3.4</commons.lang3.version>
     <commons.exec.version>1.3</commons.exec.version>
-    <interpreter.name>sh</interpreter.name>
+    <pty4j.version>0.9.3</pty4j.version>
+    <jinjava.version>2.4.0</jinjava.version>
+    <guava.version>20.0</guava.version>
   </properties>
 
+  <!-- pty4j library not in travis central (http://repo.maven.apache.org/maven2) -->
```

Comment:
It is not travis central, it is maven central repository

Reply:
Ok, I am going to modify the text description.